src/components/TodoForm.js

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+import { toast } from 'react-toastify';
+import PropTypes from 'prop-types';
+
+function TodoForm({ handleTodoChange }) {
+  const [value, setValue] = useState('');
+  const [error, setError] = useState(false);
+
+  const todo = {
+    id: Math.ceil(Math.random() * 1000),
+    title: value,
+    createdAt: new Date(),
+    updatedAt: new Date(),
+    completed: false,
+  };
```

Comment:
You should use a unique string for each id. Random numbers from 1 to 1000 can repeat. There you suggest to use something like uuid to generate unique id.

Reply:
Okay. Great thank you